scripts/libunwind_build.sh

```diff
@@ -5,12 +5,9 @@
 #
 set -euo pipefail
 
-curl -L https://download.savannah.nongnu.org/releases/libunwind/libunwind-1.5.0.tar.gz -o libunwind-1.5.0.tar.gz
-tar -xf libunwind-1.5.0.tar.gz
-# Add containers support in libunwind
```

Comment:
This is not needed anymore because the commit is included in 1.6.2?

Reply:
Yes, exactly. The commit [31459](https://github.com/libunwind/libunwind/commit/831459ee961e7d673bbd83e40d0823227c66db33) (containers support) is included in v1.6.2. 

![image](https://user-images.githubusercontent.com/11335874/195570192-b1b39d75-a4bc-4d5b-968a-57e11b55b0a3.png)


 
  

Also, this replacing `sed s/unw_ltoa/ltoa/g`  is no more needed due now we take all changes, not just from a single commit. FYI, the definition of `ltoa` has been renamed to `unw_ltoa` in commit [a52e5e7c](https://github.com/libunwind/libunwind/commit/a52e5e7c) date on 21 May 2020.

![image](https://user-images.githubusercontent.com/11335874/195572300-feb222d8-b155-455d-972d-9d6026516f16.png)